docs/user_manual/calculations.md

```diff
@@ -89,8 +89,113 @@ They can be used where approximate solutions are acceptable. The table below can
 TODO: for each of the algorithms give a brief explanation of the algorithm and in what cases this algorithm would be the prefered method. The old explanations are given, but they should be extended/improved.
 Also include the mathematics/algorithms.
 
+The nodal equations of a power system network can be written as:
+
+$$
+   \begin{eqnarray}
+      I    & = Y_{bus}V
+   \end{eqnarray}
+$$
+
+Where $I$ is the $N$ vector of source currents injected into each bus and $V$ is the $N$ vector of bus voltages. The complex power
+delivered to bus $k$ is:
+
+$$
+   \begin{eqnarray}
+      S_{k}    & =  P_k + jQ_k & = V_{k} I_{k}^{*}
+   \end{eqnarray}
+$$
+
+Power flow equations are based on solving the nodal equations above to obtain the voltage and voltage angle at each node
+and then obtaining the real and reactive power flow through the branches. The following bus types can be present in the system:
+
+- Slack bus: the reference bus with known voltage and angle; in power-grid-model referred to as the [source](./components.md#source).
+- Load bus: a bus with known $P$ and $Q$.
+- Voltage controlled bus: a bus with known $P$ and $V$.
```

Comment:
Should we also mention here that voltage controlled bus is unsupported right now?